sklearn/linear_model/_glm/glm.py

```diff
@@ -72,6 +73,7 @@ class GeneralizedLinearRegressor(RegressorMixin, BaseEstimator):
         regularization strength. ``alpha = 0`` is equivalent to unpenalized
         GLMs. In this case, the design matrix `X` must have full column rank
         (no collinearities).
+        Values should be >=0.
```

Comment:
If we are going to show ranges, I think we should be consistent with https://github.com/scikit-learn/scikit-learn/pull/21955 (When that PR gets merged)